app/uploaders/ad_hoc_data_source_uploader.rb

```diff
@@ -0,0 +1,112 @@
+###
+# Copyright 2016 - 2019 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+# encoding: utf-8
+
+require 'carrierwave/uploader/magic_mime_whitelist'
```

Comment:
Will need to be removed for Rails 5+ (CarrierWave now does this internally)

Reply:
Yes, thank you.  I had to remove it to deploy to staging.  Do we have a pattern for flagging these for removal?

Reply:
I don't... Maybe something as simple as `# FIXME Remove for Rails Upgrade`